operators/multiclusterobservability/controllers/placementrule/manifestwork_test.go

```diff
@@ -34,6 +35,10 @@ const (
 	workSize       = 13
 )
 
+func init() {
+	os.Setenv("UNIT_TEST", "true")
+}
```

Comment:
Are you sure this is a good idea? Once this env var is set, I see nothing unsetting it. 🤔

Reply:
I think this is only run in test runs and wont be set in the actual deployment

Reply:
I'm not sure wether this will run only on test because it's not in a separate `placementrule_test` package, but it shares the same `placementrule` package as the implementation.